src/core/engine/processors/kitchen_simulator.py

```diff
@@ -57,6 +59,26 @@ def assign_orders_to_available_chefs(self):
                 else:
                     self._order_send_to_cancel(order_to_be_assigned[0])
 
+    def check_for_order_completed(self):
+
+        order_id_to_be_check = self.order_manager.get_queue_from_status(
+            OrderStatus.IN_PROCESS
+        )
+
+        if order_id_to_be_check:
+            order_to_be_check = self.order_controller.get_by_id(order_id_to_be_check)
+            last_order_status_history = self.order_status_history_controller.get_last_status_history_by_order_id(
+                order_id_to_be_check
+            )
+
+            if datetime.now() > (
```

Comment:
I would prefer if you convert datetime to milliseconds and then compare in milliseconds instead

Reply:
done.